TheFirstHomeTask.playground/Contents.swift

```diff
@@ -1,3 +1,35 @@
-import UIKit
+// Sum of separately whole and separately fractional parts of numbers
+var a = 2.5
+var b = 9.7
+var c = 6.9
+var d = 8.2
 
-var greeting = "Hello, playground"
+var arrayOfValues = [a,b,c,d]
+
+var sumOfFractional: Double = 0
+var sumOfWhole: Double      = 0
```

Comment:
A few years ago I was suggested to drop prepositions in the variables naming. As we use English, we can easily drop them by just changing the order of words. For example, `valuesArray` instead of `arrayOfValues`, `fractionalSum`  instead of `sumOfFractional` and so on. There's no big difference in these namings but the naming without prepositions is shorter and sometimes it's easier to read. I really liked this suggestion and I always use it. Maybe you'll like the idea too